Description: Disable globals test
 The tests makes sense during development to check for erroneously added global
 symbols, but on a packing level we will catch the using symbols file. So we
 disable the test. Especially since the test is not able to handle -fPIE as
 default.
Author: Sebastian Ramacher <sramacher@debian.org>
Forwarded: not-needed
Last-Update: 2017-02-04

--- a/test/Makefile.am
+++ b/test/Makefile.am
@@ -3,4 +3,4 @@ compare_SOURCES = compare.c
 compare_LDADD = -lm
 
 EXTRA_DIST = regression tests compile globals
-TESTS = regression compile globals
+TESTS = regression compile
